website/src/pages/home/_UseCases.tsx

```diff
@@ -0,0 +1,105 @@
+
+import styles from './home.module.scss'
+
+interface UseCases {
+    icon: MdiReactIconComponentType
+    title: string
+    description: string
+    link?: string
+}
+
+const useCases: UseCases[] = [
+    {
+        icon: ToolsIcon,
+        title: 'Fix vulnerabilities',
+        description: 'Find, fix, and track vulnerable code quickly across your entire codebase.',
+        link: '/use-cases/vulnerabilities',
+    },
+    {
+        icon: ClockOutlineIcon,
+        title: 'Onboard developers',
+        description: 'Decrease time to first commit with codebase onboarding and knowledge sharing.',
+        link: '/use-cases/onboarding',
+    },
+    {
+        icon: CheckboxMarkedCircleOutlineIcon,
+        title: 'Resolve incidents',
+        description: 'Identify the root cause in code and fix the issue everywhere, faster.',
+        // link: '/use-cases/incidents'
+    },
+    {
+        icon: CachedIcon,
+        title: 'Promote code reuse',
+        description: 'Find existing code for reuse and contribute to a more coherent codebase.',
+        // link: '/use-cases/code-reuse'
+    },
+    {
+        icon: LaptopIcon,
+        title: 'Boost code health',
+        description: 'Improve code health with large-scale changes, and track key initiatives.',
+        // link: '/use-cases/code-health'
+    },
+]
+
+const UseCases: FunctionComponent = () => (
+    <div className="position-relative">
+        <ContentSection className="py-7">
+            <div className="text-center">
+                <h1 className="font-weight-bold">Move fast &mdash; even in big codebases</h1>
+                <a href="https://docs.sourcegraph.com">See the docs</a>
+            </div>
+
+            <div className="row max-w-800 mx-auto mt-6">
+                {useCases.map((useCase, i) => (
+                    <div
+                        key={useCase.title}
+                        className={classNames(
+                            {
+                                ['mb-6']: i !== useCases.length - 1,
+                                ['mb-0']: i === useCases.length - 1,
+                            },
+                            'col-sm-6 d-flex flex-column flex-sm-row align-items-center align-items-sm-start'
+                        )}
+                    >
+                        <useCase.icon className="text-vivid-violet w-100 max-w-50 h-auto mr-sm-2 mb-4 mb-sm-0" />
+                        <div className="text-center text-sm-left">
+                            <h4 className="font-weight-bold">{useCase.title}</h4>
+                            <div className="text-lg">
+                                <p className="m-0">{useCase.description}</p>
+                                {useCase.link && <Link to={useCase.link}>Learn more</Link>}
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </div>
+        </ContentSection>
+
+        <div
```

Comment:
Small note, looks like text should be centered on mobile from the spec

Reply:
This is already centred from `align-items-center` on mobile

Reply:
The style isn't getting applied, it seems
![image](https://user-images.githubusercontent.com/59381432/164790715-7392f37b-2093-4a28-ace3-fc40989c65c1.png)

Reply:
Oh I see what you mean! Sorry, I was looking at the line you commented on. I'll push a fix!